sql/moz-fx-data-shared-prod/search_terms_derived/sanitation_job_metadata/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+---
+friendly_name: Sanitized Firefox Search Terms
+owners:
+  - ctroy@mozilla.com
+labels:
+  incremental: true
+description: |-
+  A sanitized variant of search terms submitted to Firefox.
```

Comment:
I think this description should be updated to reflect this is a metadata table and not actual search terms data.

Reply:
Whoops!! Flipped the two descriptions. Good catch!!